docs/devel/owners.md

```diff
@@ -0,0 +1,72 @@
+# Owners files
+
+_Note_: This is a design for a feature that is not yet implemented.
+
+## Overview
+We want to establish owners for different parts of the code in the Kubernetes codebase.  These owners
+will serve as the approvers for code to be submitted to these parts of the repository.  Notable, owners
+are not necessarily expected to do the first code review for all commits to these areas, but they are
+required to approve changes before they can be merged.
+
+## High Level flow
+
+### Step One: A PR is submitted
+After a PR is submitted, a user can request the approval from the required owners.  The user requests this
+by writing:
+```
+@k8s-bot approvers
+```
+
+into a comment.  When our automation sees this comment, it will reply with an annotation that indicates
+the owners required to approve.  The annotation is a comment that is applied to the PR.
+This comment will say:
+
+```
+Approval is required from <owner-a>, <owner-b>
+```
+
+PR authors are not required to ask for approval, if for example, the reviewer is already an owner, it is
+expected that the reviewer will (eventually) grant approval without being explicitly asked to do so.
+
+### Step Two: A PR is LGTM'd
+Once a PR is reviewed and LGTM'd it is eligible for submission.  However, in order for it to be submitted
+all owners have to 'approve' the PR.  Approval comes in two forms:
```

Comment:
'all owners' is a little vague. I think you mean all owners listed by the bot above, which is really just owners over the parts of the repository that are being changed.